### PR TITLE
Bug 1920674: Fix missingKey errors

### PR DESCRIPTION
**Pull request by rebeccaalpert**
The bindings namespace had various missingKey errors due to usage of i18next.t. I updated the code to use the correct approach.

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1920674.

### CODE REVIEW

File: frontend/public/components/RBAC/bindings.jsx [hunk 1]
```diff
@@ -522,6 +500,27 @@ class BaseEditRoleBindingWithTranslation extends React.Component {
     const { fixed, saveButtonText, t } = this.props;
     const RoleDropdown = kind === 'RoleBinding' ? NsRoleDropdown : ClusterRoleDropdown;
     const title = `${this.props.titleVerb} ${kindObj(kind).label}`;
+    const bindingKinds = [
+      {
+        value: 'RoleBinding',
+        title: t('bindings~Namespace RoleBinding'),
```
Comment (spadgett) [inline frontend/public/components/RBAC/bindings.jsx:506]:
```suggestion
        title: t('bindings~Namespace role binding (RoleBinding)'),
```

File: frontend/public/components/RBAC/bindings.jsx [hunk 2]
```diff
@@ -522,6 +500,27 @@ class BaseEditRoleBindingWithTranslation extends React.Component {
...
+        desc: t(
+          'bindings~Grant the permissions to a user or set of users within the selected namespace.',
+        ),
+      },
+      {
+        value: 'ClusterRoleBinding',
+        title: t('bindings~Cluster-wide RoleBinding (ClusterRoleBinding)'),
```
Comment (spadgett) [inline frontend/public/components/RBAC/bindings.jsx:513]:
```suggestion
        title: t('bindings~Cluster-wide role binding (ClusterRoleBinding)'),
```

**Review by spadgett — COMMENTED** [review]
This isn't new, but I have one suggestion on the text to be the different binding types more clear.

**Comment by rebeccaalpert** [comment]
All set @spadgett.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**